corva/models/stream.py

```diff
@@ -112,15 +166,19 @@ def filter(
                 by_timestamp=by_timestamp,
                 by_depth=by_depth,
                 last_timestamp=last_timestamp,
-                last_depth=last_depth
+                last_depth=last_depth,
```

Comment:
Why are we adding trailing commas to things?

Reply:
I use black locally for consistent code formatting

Reply:
That's fine, but please update the whole SDK to use Black during CI and in the Readme so we don't have inconsistent methods for lint.

Reply:
Yes, I'm planning to add isort, black and autoflake to CI in the future for consistent code formatting

Reply:
Thank you!